packages/blocks/code/src/app.tsx

```diff
@@ -84,6 +84,7 @@ export const App: BlockComponent<BlockEntityProperties> = ({
 
   const copyToClipboard = async () => {
     try {
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- upstream typings assume that Clipboard is always present
       if (navigator.clipboard) {
```

Comment:
Is it possible to overwrite the typings in our environment to say maybe it's not? Ensures we check this everywhere